dask/multiprocessing.py

```diff
@@ -236,12 +261,16 @@ def get(
     return result
 
 
-def initialize_worker_process():
+def initialize_worker_process(user_initializer=None, override=False):
     """
     Initialize a worker process before running any tasks in it.
     """
     # If Numpy is already imported, presumably its random state was
     # inherited from the parent => re-seed it.
-    np = sys.modules.get("numpy")
-    if np is not None:
-        np.random.seed()
+    if user_initializer is None or not override:
```

Comment:
Interesting this code makes me think that `replace_initializer` might not be the right name either. Maybe something like `disable_default_initializer` or `use_default_initializer` or similar?

```suggestion
    if user_initializer is None or not replace_initializer:
```

Reply:
I'm less in favor of `use_default_initializer` than the other names, since it's somewhat ambiguous in its meaning of "use".  I mean, does it mean "use only the default" or "use the default including the user-defined initializer".

`disable_default_initializer` is the best, IMO.

Reply:
Sure not attached to any of those names either. So if you can think of a better one, happy to go with that